ISSUE_TEMPLATE/tracking.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Tracking Issue
+about: Open an issue to track progress toward implementation of a merged RFC
+title: '[RFC #<INSERT RFC NUMBER - e.g., 0099>] - <INSERT RFC TITLE>'
+labels: type/tracking
+assignees: ''
+
+---
+
+[RFC #<INSERT RFC NUMBER - e.g., 0099>](https://github.com/buildpacks/rfcs/blob/main/text/<INSERT FILENAME>) - <INSERT RFC TITLE>
+
```

Comment:
are we calling out `pack` / `lifecycle` explicitly or just want a generic thing + spec/documentation? For instance, I can imagine changes for `libcnb` too or really any repo we have.

Reply:
Makes sense... we could add a placeholder for `<insert repo here>` and folks could just delete it if it's not relevant.

Reply:
WDYT of 5faf18e ?

Reply:
Or maybe... I feel like most changes will touch spec + lifecycle + ??? + docs - where ??? could be pack, libcnb, or some combination of the two. I'm probably biased toward pack because I've worked on that repo and not libcnb 🙃 

I am happy with whatever everyone feels is the most useful.

Reply:
I would say that any changes to buildpack API require changes in libcnb.

Reply:
Please see df8e81c - added libcnb so it's on equal footing with pack.